.github/workflows/wheels.yml

```diff
@@ -0,0 +1,38 @@
+name: Build
+
+# To publish on every pull request
+# on: [push, pull_request]
+# To publish when a (published) GitHub Release is created, use the following:
+on:
+  push:
+  pull_request:
+  release:
+    types:
+      - published
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-10.15]
```

Comment:
I think I understand this that the ubuntu one will use docker to install all the various python flavors using "manylinux".  Whatever that means.  :)  

And we don't need to do the multiple python versions in the matrix, since it will automatically spawn all of those?

Reply:
exactly, the linux matrix entry will just use a series of containers to build the various versions of the wheel. 

The manylinux image is essentially a barebone linux with maximal compatibilty with other linux variants. The script builds the wheel in that environment, and also uses a magical tool to package in the wheel all shared libraries needed  for the execution. So that the final binary is almost completely self contained, only retain some specificity against the python version and platform used to compile it.